ui/app/AppLayouts/Communities/panels/OverviewSettingsFooter.qml

```diff
@@ -0,0 +1,141 @@
+import QtQuick 2.15
+import QtQml 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+Control {
+    id: root
+    property bool isControlNode: true
+    property int loginType: Constants.LoginType.Password
+    property string communityName: ""
+
+    signal primaryButtonClicked
+    signal secondaryButtonClicked
+
+    QtObject {
+        id: d
+        
+        readonly property real verticalBreakPoint: 950
```

Comment:
What about using something like a `Flow` instead of hardcoding the break point?

Reply:
`Flow` was my original choice but it didn't work nicely. Since it's a positioner, it's missing the ability to automatically fill in the available area and to dynamically resize the items.